include/RAJA/policy/desul/atomic.hpp

```diff
@@ -26,6 +26,27 @@ using raja_default_desul_scope = desul::MemoryScopeDevice;
 namespace RAJA
 {
 
+RAJA_SUPPRESS_HD_WARN
+template <typename AtomicPolicy, typename T>
+RAJA_HOST_DEVICE
+RAJA_INLINE T
+atomicLoad(AtomicPolicy, T volatile *acc) {
```

Comment:
```suggestion
atomicLoad(AtomicPolicy, T volatile *acc)
{
```

Format consistency with other routine.

Reply:
Fixed.